packages/react-components/react-aria/library/src/button/useARIAButtonShorthand.ts

```diff
@@ -14,10 +14,14 @@ import type { ARIAButtonProps, ARIAButtonSlotProps, ARIAButtonType } from './typ
  * for multiple scenarios of shorthand properties. Ensuring 1st rule of ARIA for cases
  * where no attribute addition is required.
  */
-// eslint-disable-next-line @typescript-eslint/no-deprecated
-export const useARIAButtonShorthand: ResolveShorthandFunction<ARIAButtonSlotProps> = (value, options) => {
+// eslint-disable-next-line @typescript-eslint/no-deprecated, @typescript-eslint/no-explicit-any
+export const useARIAButtonShorthand: ResolveShorthandFunction<any> = ((value, options) => {
```

Comment:
- why are we using `any` as generic ?
- you can remove the explicit annotation as you are using assertion already
```suggestion
export const useARIAButtonShorthand = ((value, options) => {
```